service/controller/v25/resource/tccp/error.go

```diff
@@ -30,6 +31,28 @@ func IsAlreadyExists(err error) bool {
 	return false
 }
 
+var alreadyTerminatedError = &microerror.Error{
+	Kind: "alreadyTerminatedError",
+}
+
+// IsAlreadyTerminated asserts alreadyTerminatedError.
+func IsAlreadyTerminated(err error) bool {
+	c := microerror.Cause(err)
+
+	aerr, ok := c.(awserr.Error)
+	if ok {
+		if aerr.Code() == "IncorrectInstanceState" {
```

Comment:
> {"caller":"github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/controller/resource/retryresource/resource_wrapper.go:98","controller":"aws-operator","event":"delete","level":"warning","loop":"19","message":"retrying due to error","object":"/apis/provider.giantswarm.io/v1alpha1/namespaces/default/awsconfigs/38z4i","resource":"tccpv25","stack":"[{/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/controller/resource/retryresource/resource_wrapper.go:91: } {/go/src/github.com/giantswarm/aws-operator/service/controller/v25/resource/tccp/delete.go:28: } {/go/src/github.com/giantswarm/aws-operator/service/controller/v25/resource/tccp/resource.go:188: } {IncorrectInstanceState: The instance 'i-0b26c88f3546aefee' must be in a 'running', 'pending', 'stopping' or 'stopped' state for this operation.\n\tstatus code: 400, request id: 9d57b148-da42-49bc-ae7f-c121f7c9e38d}]","time":"2019-03-22T14:03:18.216824+00:00","underlyingResource":"tccpv25","version":"109386904"}

Reply:
How about adding this as a comment?

```
	IncorrectInstanceState: The instance 'i-0b26c88f3546aefee' must be in a 'running', 'pending', 'stopping' or 'stopped' state for this operation.
```

Reply:
See https://github.com/giantswarm/aws-operator/pull/1524.